assets/js/blocks/cart-checkout/cart/frontend.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { withRestApiHydration } from '@woocommerce/block-hocs';
```

Comment:
Not sure why was this necessary since we have side effects enabled, is it because the whole project is treated as a single package? would it be worth making each folder a separate package or just using relative paths?
cc @nerrad

Reply:
It was including all HOCs imported from `index.js`, but after building it with `ENV='production'` it seems to work as expected, so false alarm. I reverted this change. Thanks for catching it @senadir & @nerrad.